cspell.json

```diff
@@ -224,6 +224,7 @@
         ".git/**",
         "**/.gitignore",
         ".vscode/**",
-        "**/.idea/**"
+        "**/.idea/**",
+        "hermes/wasm/*/**/**"
```

Comment:
I think it could use `hermes/wasm/*/**/**` as the extra `/**` is unnecessary. The point of `hermes/wasm/*/**` here is to exclude the check from checking content inside `crates` as it is a forked code. But the direct content inside `hermes/wasm` is needed to be checked, e.g., `README.md`. Or can you confirm that we just need to exclude only `hermes/wasm/crates/wasi-component-adapter`?

Reply:
Double checked, seems like that we need to exclude `hermes/wasm/crates/wasi-component-adapter` and `hermes/wasm/crates/wasi` other stuff should be in place I think